README.md

```diff
@@ -62,10 +67,27 @@ Outstanding issues and requests can be tracked and visualised on our Github Proj
 [Documentation](https://github.com/Mafia-Hands/Mafia/projects/3)
 
 ## Contributing
-If you would like to contribute to this project, please make sure to follow our [Contributing-Guidlines](https://github.com/Mafia-Hands/Mafia/wiki/Contributing-Guidelines).
+
+If you would like to contribute to this project, please make sure to follow our [Contributing-Guidlines](https://github.com/Mafia-Hands/Mafia/wiki/Contributing-Guidelines)s.
```

Comment:
If you would like to contribute to this project, please make sure to follow our [Contributing-Guidelines](https://github.com/Mafia-Hands/Mafia/wiki/Contributing-Guidelines).